dhcpv6/client6/client.go

```diff
@@ -177,6 +177,10 @@ func (c *Client) sendReceive(ifname string, packet dhcpv6.DHCPv6, expectedType d
 		adv, err = dhcpv6.FromBytes(buf[:n])
 		if err != nil {
 			// skip non-DHCP packets
+			//
+			// TODO: It also skips DHCP packets with any errors (for example
```

Comment:
is this TODO for this PR or a future one ? It would be useful to make an issue for this if it won't be addressed in this PR

Reply:
Created an issue: https://github.com/insomniacslk/dhcp/issues/341